recipes/opentelemetry-cpp/all/conanfile.py

```diff
@@ -67,24 +66,37 @@ class OpenTelemetryCppConan(ConanFile):
     short_paths = True
 
     @property
-    def _minimum_cpp_standard(self):
+    def _min_cppstd(self):
+        if self.options.with_abseil and Version(self.dependencies["abseil"].ref.version) >= "20230125":
+            return 14
         return 11
 
-    def export_sources(self):
-        export_conandata_patches(self)
+    @property
+    def _compilers_minimum_version(self):
+        if self._min_cppstd == 14:
+            return {
+                "gcc": "6",
+                "clang": "5",
+                "apple-clang": "10",
+                "Visual Studio": "16",
+                "msvc": "192",
+            }
+        else:
+            return {
+                "Visual Studio": "16",
+                "msvc": "192",
+            }
 
     def config_options(self):
         if self.settings.os == "Windows":
             self.options.rm_safe("fPIC")
+        if Version(self.version) >= "1.10":
+            del self.options.with_jaeger
 
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
 
```

Comment:
```suggestion
    if self.options.with_otlp != "deprecated":
        self.output.warning(f"{self.ref}:with_otlp option is deprecated, do not use anymore. Please, consider with_otlp_grpc or with_otlp_http instead.")
```

https://github.com/conan-io/conan-center-index/blob/f174071b1e873360c644b401e61a66bd3c8f9aac/docs/faqs.md#can-i-remove-an-option-from-a-recipe